### PR TITLE
docs: refactoring, metrics update

### PR DESCRIPTION
**Pull request by Kaspiman**
## Summary by CodeRabbit

- **Documentation Updates**
    - Reordered sections in PHP Worker documentation for better clarity.
    - Corrected numbering in the Config plugin documentation.
    - Enhanced memory metrics documentation to distinguish between single and total workers' usage.
    - Introduced new metrics for tracking worker states.
    - Clarified the description of worker's maximum lifespan in the worker pool configuration.

### CODE REVIEW

File: lab/metrics.md [hunk 1]
```diff
@@ -62,11 +64,10 @@ http:
 
 The HTTP metrics provided by the metrics plugin include:
 
-- `rr_http_requests_queue` - Total number of queued requests waiting for a worker.
-- `rr_http_uptime_seconds` - Plugin uptime in seconds.
-- `rr_http_no_free_workers_total` - Total number of NoFreeWorkers errors.
```
Comment (Kaspiman) [inline lab/metrics.md:67]: this metric does not exist

File: lab/metrics.md [hunk 2]
```diff
@@ -62,11 +64,10 @@ http:
...
 - `rr_http_request_total` - Total number of handled HTTP requests after server restart.
```
Comment (Kaspiman) [inline lab/metrics.md:67]: Frequently used elements have been moved higher

File: SUMMARY.md [hunk 3]
```diff
@@ -11,13 +11,12 @@
 * [Upgrade and Compatibility](intro/compatibility.md)
 
 ## 👷 PHP Worker
-
-* [Dynamic Workers Scaling](php/scaling.md)
-* [Developer mode](php/developer.md)
-* [Environment](php/environment.md)
-* [Debugging](php/debugging.md)
 * [Worker](php/worker.md)
```
Comment (Kaspiman) [inline SUMMARY.md:14]: Frequently used elements have been moved higher

File: lab/metrics.md (hunk 1 above)
Comment (rustatian) [reply lab/metrics.md:67]: It exists -> https://github.com/roadrunner-server/prometheus/blob/master/plugin.go#L58, but w/o `rr_http_`.

**Review by rustatian — CHANGES_REQUESTED** [review]

File: lab/metrics.md (hunk 1 above)
Comment (Kaspiman) [reply lab/metrics.md:67]: Hmmm.... Sure.

Comment (rustatian) [reply lab/metrics.md:67]: It exists within the `rr_http` namespace, w/o `rr_http_` is the mistake.

Comment (Kaspiman) [reply lab/metrics.md:67]: So, no deletions from me

**Review by rustatian — APPROVED** [review]
LGTM, thanks @Kaspiman 👍

**Pull request merged**